tools/block-generator/generator/generate_apps.go

```diff
@@ -69,9 +109,10 @@ func countEffects(actual TxTypeID) uint64 {
 	return cnt
 }
 
+// TODO: Can this be removed? The inner txns are already marked as "inner" so no nened to aggregate them.
```

Comment:
This also computes the "sibling" transactions. But if we don't care enough about these we could remove.

Reply:
BTW. Currently there is a bug and CumulativeEffects are only being reported for one of the blocks - so they aren't actually cumulative.

Reply:
Another thought. Frankly, the number "sibling txns" doesn't seem particularly informative. Possibly more informative: the number of "atomic txns".

Reply:
Whats the difference between sibling and inner transactions? txgroup level?

The CSV output is currently what I'm using to compare scenarios, so I haven't been using these functions.

As a unit of work, how many transactions (regardless of type) and how much total data are probably the most relevant to Conduit.

Reply:
I'll remove this comment, maybe this data will be useful in the future.